src/components/views/rooms/RoomPreviewBar.js

```diff
@@ -33,16 +33,22 @@ module.exports = React.createClass({
 
         // If invited by 3rd party invite, the email address the invite was sent to
         invitedEmail: React.PropTypes.string,
-        canJoin: React.PropTypes.bool,
+
+        // A standard client/server API error object. If supplied, indicates that the
+        // caller was unable to fetch details about the room for the given reason.
+        error: React.PropTypes.object,
+
         canPreview: React.PropTypes.bool,
         spinner: React.PropTypes.bool,
         room: React.PropTypes.object,
+
+        // The alias that was used to access this room, if appropriate
```

Comment:
Might also be worth recording what it is used for (ie, sticking into error messages).

I wondered if it might actually be better called roomName (since, from the POV of `RoomPreviewBar` it's just a name for the room), but that is probably going too far and is too confusing with the *actual* room name.